### DISCUSSION TITLE
Copilot with ozone code?

### DISCUSSION THREAD

**Discussion by michelsumbul** [apache/ozone#5147] [General]
Hey guys,
I was wondering if anyone have try Copilot from girhub to develop on the ozone project?
Does it help to develop quicker or is it rubbish on a big project like this one?
I'm curious if copilot would have recommended on the current code.

I say copilot but if someone used another generative Ai for coding, I'm interested on the experience

Michel

**Comment by arp7** [comment]
Topical announcement from the ASF: https://www.apache.org/legal/generative-tooling.html
